app/main.py

```diff
@@ -1,3 +1,24 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    data_dict = {}
+
+    def inner(*args) -> Callable:
```

Comment:
It does not return Callable it can return Any, because result of function can be any, we can not know it